.github/README.md

```diff
@@ -0,0 +1,27 @@
+# Bridge-In-Tech (android)
```

Comment:
> (android)

This repo is actually the 'backend' of the Bridge-In-Tech system, @Menina294. The 'android' is the frontend, and it's another [repo](https://github.com/anitab-org/bridge-in-tech-android) of bridge-in-tech. Can you please fix this 😉 ?

Reply:
+1